packages/@react-spectrum/autocomplete/test/SearchAutocomplete.test.js

```diff
@@ -3228,7 +3228,9 @@ describe('SearchAutocomplete', function () {
 
         let listbox = getByRole('listbox');
         expect(listbox).toBeVisible();
-        expect(screen.getAllByRole('log')).toHaveLength(2);
+        expect(announce).toHaveBeenCalledTimes(2);
```

Comment:
these tests were honestly checking something silly, so I've updated them to check useful information